app/app.go

```diff
@@ -30,16 +30,8 @@ func NewApp(params AppParams) (*cli.App, error) {
 	}
 
 	var commands []*cli.Command
+	commands = append(commands, params.Commands...)
```

Comment:
nit it seems like this can be simplified to be `app.Commands = params.Commands` ?